test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,167 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.tag.Tag;
+
+public class UtilsTest {
+    private static Object nullObject = null;
+    private static Tag tag1;
+    private static Tag tag1Copy;
+    private static Tag tag2;
+
+    @Before
+    public void setup() {
+        tag1 = buildTag("tag1");
+        tag1Copy = buildTag("tag1");
+        tag2 = buildTag("tag2");
+    }
+    
+    private static Tag buildTag(String tagName) {
+        try {
+            return new Tag(tagName);
+        } catch (IllegalValueException e) {
+            throw new RuntimeException("test tag name should be valid", e);
+        }
+    }
+
+    @Test
+    public void isAnyNull_emptyInput_returnsFalse() {
+        assertFalse(Utils.isAnyNull());
+    }
```

Comment:
`isAnyNull` is a simple method. Can be tested using one test method. Cases to test:
No nulls: 
* any non-empty list, 
* an empty list
With nulls: 
* non empty list with just one at the beginning (to check for bugs missing the first item in the list), 
* list with multiple objects, couple of nulls in the middle
* non empty list with one null as the last element (to check for bugs missing last item in the list)